website/docs/r/container_group.html.markdown

```diff
@@ -132,6 +134,16 @@ The following arguments are supported:
 
 ---
 
+An `identity` block supports the following:
+
+* `type` - (Required) The Managed Service Identity Type of this container group. Possible values are `SystemAssigned` (where Azure will generate a Service Principal for you), `UserAssigned` where you can specify the Service Principal IDs in the `identity_ids` field, and `SystemAssigned, UserAssigned` which assigns both a system managed identity as well as the specified user assigned identities.
```

Comment:
(as above) can we confirm what this third value (`SystemAssigned, UserAssigned`) should be?

Reply:
'SystemAssigned, UserAssigned' is correct... the validation function is wrong. Correcting now. See example in the Azure docs here:

https://docs.microsoft.com/en-us/azure/container-instances/container-instances-managed-identity#enable-managed-identity-using-yaml-file